components/activity_log_modal/activity_log_modal.test.jsx

```diff
@@ -21,18 +20,6 @@ describe('components/ActivityLogModal', () => {
         locale: General.DEFAULT_LOCALE,
     };
 
-    test('should match snapshot', () => {
-        const wrapper = shallow(
-            <ActivityLogModal {...baseProps}/>,
-        );
-        expect(wrapper).toMatchSnapshot();
-        expect(wrapper.find(LoadingScreen).exists()).toBe(false);
```

Comment:
I don't think we need to completely remove the snapshot tests, we can just update the snapshots and not worry about checking if the `LoadingScreen` is there.